ui/app/AppLayouts/Chat/views/ChatContextMenuView.qml

```diff
@@ -221,7 +221,7 @@ StatusPopupMenu {
                 close()
             }
             onConfirmButtonClicked: {
-                if(root.isCommunityChat || root.chatType === Constants.chatType.oneToOne)
```

Comment:
This was a mistake, only the community needs to call `deleteChat`, all other chat types use `leaveChat`